crates/ruff/src/rules/flake8_annotations/rules.rs

```diff
@@ -420,10 +420,7 @@ fn is_none_returning(body: &[Stmt]) -> bool {
     for expr in visitor.returns.into_iter().flatten() {
         if !matches!(
             expr.node,
```

Comment:
Or even `expr.node.is_constant_none()`?

Reply:
I think it's fine the way it is. An alternative would be to do `expr.as_constant().map_or(false, |constant| constant.is_none())` but that's at least as verbose.